CMS/static/scss/global.scss

```diff
@@ -218,3 +218,19 @@ a[target=_blank] {
   }
 }
 
+a.skip-main {
+  position: absolute;
+  left: -999px;
+  width: 1px;
+  height: 1px;
+  top: auto;
+}
+
+a.skip-main:focus {
```

Comment:
could make use of scss nesting and the '&' thing here?

Reply:
Did it. Thank you!

Reply:
Could you give me a thumbs up if it's good to go? @VivianAllen

Reply:
👍 from me